common/src/main/java/whocraft/tardis_refined/common/tardis/themes/ShellTheme.java

```diff
@@ -20,6 +20,8 @@ public class ShellTheme implements Theme {
     /** Global instance of the Shell Theme custom registry created by Tardis Refined*/
     public static final Registry<ShellTheme> SHELL_THEME_REGISTRY = SHELL_THEMES.getRegistry();
 
+    public static final RegistrySupplierHolder<ShellTheme, ShellTheme> HALF_BAKED = registerShellTheme("half_baked");
```

Comment:
If this is meant to be the default theme we could add a comment to specify that so that we don't misuse/remove it.